src/main/java/de/zalando/aruha/nakadi/service/EventStream.java

```diff
@@ -101,7 +99,7 @@ public void streamEvents() {
                     currentBatch = emptyBatch();
                     batchStartTime = currentTimeMillis();
 
-                    outputStream.write(BATCH_SEPARATOR.getBytes());
+                    outputStream.write(BATCH_SEPARATOR.getBytes(UTF8));
```

Comment:
`getBytes()` uses the default charset which is determined by the jvm i.e. the underlying os. Hence, if `byte[] bytes = s.getBytes()` is called on a windows machine, the `bytes` are send to a linux machine and `new String(bytes)` is called, it might fail.